config/playground/demo/observability/otel_demo/jaeger.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jaeger-config
+  namespace: otel-demo
+data:
+  COLLECTOR_OTLP_ENABLED: "true"
+  METRICS_STORAGE_TYPE: "prometheus"
+  SPAN_STORAGE_TYPE: "opensearch"
+  ES_TAGS_AS_FIELDS_ALL: "true"
+  ES_USERNAME: "admin"
+  ES_PASSWORD: "admin"
+  ES_TLS_SKIP_HOST_VERIFY: "true"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jaeger
+  namespace: otel-demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jaeger
+  template:
+    metadata:
+      labels:
+        app: jaeger
+    spec:
+      containers:
+        - name: jaeger
+          image: jaegertracing/jaeger-collector:latest
+          command:
+            - "--metrics-backend=prometheus"
+            - "--es.server-urls=https://opensearch-cluster-leader:9200"
+            - "--es.tls.enabled=true"
+          envFrom:
+            - configMapRef:
+                name: jaeger-config
+          resources:
+            limits:
+              memory: 300Mi
+          ports:
+            - containerPort: 14267
+            - containerPort: 14268
+            - containerPort: 14269
+            - containerPort: 14250
+            - containerPort: 4317
+            - containerPort: 9411
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: jaeger
+spec:
+  selector:
+    app: jaeger
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 14268  # Update with the correct targetPort for your service
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jaeger-agent
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jaeger-agent
+  template:
+    metadata:
+      labels:
+        app: jaeger-agent
+    spec:
+      containers:
+        - name: jaeger-agent
```

Comment:
same as above, container crashed as cannot find $Path, the following works for me 

````
    spec:
      containers:
        - args:
            - "--reporter.grpc.host-port=jaeger:14250"
          name: jaeger-agent
          image: jaegertracing/jaeger-agent:latest
          env:
            - name: SPAN_STORAGE_TYPE
              value: "opensearch"
````